### PR TITLE
Task1

### PR DESCRIPTION
**Pull request by horaanchal**
PR raised as instructed!!

### CODE REVIEW

**Review by amandeeprikhi — DISMISSED** [review]
LGTM

**Review by amandeeprikhi — CHANGES_REQUESTED** [review]
- Cannot really comment on each file individually as you seem to have imported the Bootstrap and Font Awesome libraries locally(approx. 6500 new files) and pushed them with your commit. Please can you remove them from the code base.
- Apart from this the Factorial button in Calculator does not seem to work.
- Also, this repo contains 2 readme.md files in the root directory i.e., bootcamp-jan-2020 folder. Please remove the Readme.md that you have created that contains your comments ('my name is anchal......'). Please move that file to your own folder. Essentially all the changes that you make should be in your folder only.
- In the task1 folder for the mypage.html you seemed to have used an absolute path for the icon file.

- **Good use of eval function in order to evaluate the expression string.**

File: Anchal_Hora/Calc/calc.html [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    
+    <div class="container-grid">
+        <div class="output"><form name="calc"><input type="text" name="display" id="scn" size="20" readonly></form></div>
+
+        <div class="buttons">
+            <p>
+                <input type="button" value="x!" class="other_opr" onclick=fact();>
+                <input type="button" value="C" class="other_opr">
+                <input type="button" value="CE" class="other_opr" id="ce_btn" onclick="calc.display.value=''">
+                <button class="major_opr" onclick="calc.display.value+='/'"><i class='fa fa-divide '></i></button>
```
Comment (nikhilrstg18) [inline Anchal_Hora/Calc/calc.html]: Violating Single Responsibility Principle

**Review by nikhilrstg18 — CHANGES_REQUESTED** [review]
Missing SOLID DRY KISS in code
Some of the code is copy-pasted 
Why there are 6751 files ?? can you add a comment on each one what's there purpose

File: Anchal_Hora/Calc/calc.html (hunk 1 above)
Comment (horaanchal) [reply Anchal_Hora/Calc/calc.html]: okay!

**Comment by horaanchal** [comment]
I have removed the external css file from the profile work now, thus the issue of Single Responsibility Principle is not violating. 
Also the SOLID DRY KISS is implemented.

File: Anchal_Hora/Calc/calc.html (hunk 1 above)
Comment (nikhilrstg18) [reply Anchal_Hora/Calc/calc.html]: I still see JS in HTML, not resolved

**Review by nikhilrstg18 — CHANGES_REQUESTED** [review]
I still see scope of improvement added feedback for the same, again reply on each PR review comment
if you are are using cdn for bootstrap styles and fonts, please get rid of those 6751 files from the PR
rest changes looks good to me

**Comment by horaanchal** [comment]
Removed files as you said and also resolved other issues as well.

**Comment by horaanchal** [comment]
Everything is sorted now..

**Comment by horaanchal** [comment]
I won't be able to comment on each PR review, may be I am not allowed to.
Please approve my PR now.

**Review by nikhilrstg18 — APPROVED** [review]
looks good to me

**Review by amandeeprikhi — APPROVED** [review]
LGTM

**Comment by vishivish18** [comment]
@horaanchal  Please fix the conflicts and ask for as merge

**Comment by horaanchal** [comment]
@vishivish18 Please merge it now.

**Pull request merged**